pyseq.py

```diff
@@ -86,11 +86,17 @@ def __init__(self, item):
         log.debug('adding %s' % item)
         self.item = item
         self.__path = getattr(item, 'path', os.path.abspath(str(item)))
-        self.__dirname = os.path.dirname(str(item))
+        self.__dirname = os.path.dirname(self.__path)
```

Comment:
Maybe someone understands why, but dirname was missing on the last item when using str(item), now when using self.__path it is always present...
and wasnot there a dirname on the sequence object too ?